quantize.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+
+"""Script to execute quantization on a given model."""
+
+import subprocess
+import argparse
+import sys
+import os
+
+
+def main():
+    """Parse the command line arguments and execute the script."""
+
+    parser = argparse.ArgumentParser(
+        prog='Quantization Script',
+        description='This script quantizes a model.'
+    )
+
+    parser.add_argument("models", nargs='+', dest='models')
+    parser.add_argument(
+        '-r', '--remove-16', action='store_true', dest='remove_f16',
+        help='Remove the f16 model after quantizing it.'
+    )
+
+    args = parser.parse_args()
+
+    for model in args.models:
+
+        model_path = os.path.join("models", model, "ggml-model-f16.bin")
+
+        for i in os.listdir(model_path):
+            subprocess.run(
+                ["./quantize", i, i.replace("f16", "q4_0"), "2"],
```

Comment:
also the `quantize` script isn't necessarily in the current directory (example: CMake builds).

would be useful if it is a parameter (with a default at `./`) so we could point it elsewhere easily too.

Reply:
Functionality added; you also gave me the idea of adding an argument that lets the user select the directory where the models are located, which defaults to `os.path.join(os.getcwd(), "models")`!